library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -500,6 +500,8 @@ nav#subSites{
 			&.is-current{
 				position: relative;
 				margin-right: -1px;
+				font-size: 1.2em;
+				border-right: $tool-menu-item-selected-border-right;
```

Comment:
This technique does not match the technique used for the left highlight border, which is placed with ":before" to avoid the diagonal corner border issue when applying a border to only one edge. Although you won't be able to also use the ":before", there is the ":after" that you could use.

However, the ":after" seems to be used for applying the "eye" icon to indicate a hidden tool. I think this icon should probably be added to the markup, which would free up the ":after" for your implementation. (There should probably be some type of ARIA attribute added to the hidden tools to indicate the tools are hidden, right? Also that hidden tool text colour contrast isn't WCAG 2.0 either, but these seem like other issues...)

Your implementation also misses the hover effect non-current tools.